R/model.R

```diff
@@ -925,11 +925,14 @@ as_generator.default <- function(x) {
 }
 
 as_generator.tensorflow.python.data.ops.dataset_ops.Dataset <- function(x) {
-  python_path <- system.file("python", package = "tfdatasets")
+  python_path <- system.file("python", package = "keras")
```

Comment:
That was a big oversight! Glad you caught that one.

Reply:
yeah well that only resulted in a warning from `normalizePath`, but it still worked until 1.13, but for 1.13 we need the `DatasetV2` thing...